cheatsheets/Java_Security_Cheat_Sheet.md

```diff
@@ -561,12 +563,14 @@ public class App {
         // Read the keyset into a KeysetHandle.
         KeysetHandle handle =
         TinkJsonProtoKeysetFormat.parseKeyset(
-            new String(Files.readAllBytes( Paths.get("/home/josh/aead_test_keyset.json")), UTF_8), InsecureSecretKeyAccess.get());
+            new String(Files.readAllBytes( Paths.get("/home/fred/aead_test_keyset.json")), UTF_8), InsecureSecretKeyAccess.get());
 
         String message = "This message to be encrypted";
         System.out.println(message);
 
-        String metadata = "Info about message";
+        // Add some relevant context about the encrypted data that should be verified
+        // on decryption
+        String metadata = "Sender: fredbloggs";

```

Comment:
I personally would have went with something like "/home/pete/aead_test_keyset.json" above and
```java
       String metada = "Sender: Pete Moss";
```
because it's hard to hard for me to resist inserting puns, but that's just me. But this is better than the original. More typically, the 'Sender' metadata would be either DN of a user from (say) AD or an email address, but this works.

Reply:
Not going with a pun but I made it an email address